.changeset/chilly-eggs-train.md

```
@@ -0,0 +1,15 @@
+---
+'@mastra/core': patch
+'@mastra/schema-compat': patch
+---
+
+Fixed agent network mode failing with "Cannot read properties of undefined" error when tools or workflows don't have an `inputSchema` defined.
+
+**@mastra/core**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)